VsGallery.WebService/Controllers/UploadController.cs

```diff
@@ -48,8 +49,16 @@ public IHttpActionResult AddOrUpdateVsix()
                 // Save the uploaded file to "UploadedFiles" folder
                 httpFile.SaveAs(fileSavePath);
 
+                var newFileName = httpFile.FileName.Split(new[] {".vsix"}, StringSplitOptions.None)[0] +
+                                  DateTime.UtcNow.ToString("yyyyMMdd").Replace("-", "");
+
+                var iteration = Directory.EnumerateFiles(_configuration.VsixStorageDirectory)
+                                    .Count(x => x.Split('\\').Last().StartsWith(newFileName,
+                                        StringComparison.OrdinalIgnoreCase)) + 1;
```

Comment:
Hi Could you explain to me why you need this change. This can be circumvented by having the version as a part of the package name I would think...

Reply:
Hi sorry for the delay I didn't see your reply. In production it works fine as the version is appended to the package name. However, this causes issues in our feature branch deployment as we have to manually increment the revision number each time we build and deploy for the rest of the team to use when, ideally, we would not change the actual version number as the version itself hasn't actually changed.